include/mbedtls/pk.h

```diff
@@ -611,20 +550,27 @@ int mbedtls_pk_verify(mbedtls_pk_context *ctx, mbedtls_md_type_t md_alg,
  *
  * \note            Performs the same job as \c mbedtls_pk_verify(), but can
  *                  return early and restart according to the limit set with
- *                  \c mbedtls_ecp_set_max_ops() to reduce blocking for ECC
+ *                  \c psa_interruptible_set_max_ops() to reduce blocking for ECC
  *                  operations. For RSA, same as \c mbedtls_pk_verify().
  *
+ * \note            See the note on \c mbedtls_pk_verify() for which algorithm is
+ *                  used. If you want to select a specific signature algorithm
+ *                  since the is no restartable equivalent of \c
+ *                  mbedtls_pk_verify_ext(), you'll want to import the key into
+ *                  PSA using \c mbedtls_pk_import_into_psa() and then call PSA
+ *                  interruptible functions directly.
```

Comment:
Given that this function is only useful for ECDSA, and there is only one ECDSA verification algorithm, I think this whole note is useless. For RSA, there's already a sentence above that says it's the same as `mbedtls_pk_verify`.